src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -25,37 +31,53 @@ object LocalBackend {
 }
 
 
+/**
+ * Handles both local Docker runs as well as local direct command line executions.
+ * Or at least it will.
+ */
 class LocalBackend extends Backend {
-
   /**
    * Executes the specified command line, using the supplied lookup function for expression evaluation.
    * Returns a `Map[String, Try[WdlValue]]` of output names to values.
    */
-  override def executeCommand(commandLine: String, call: Call, taskOutputs: Seq[TaskOutput], scopedLookupFunction: ScopedLookupFunction): Map[String, Try[WdlValue]] = {
+  override def executeCommand(commandLine: String, workflowDescriptor: WorkflowDescriptor, call: Call, taskOutputs: Seq[TaskOutput], scopedLookupFunction: ScopedLookupFunction): Map[String, Try[WdlValue]] = {
     import LocalBackend._
 
-    val (stdoutFile, stdoutWriter) = FileUtil.tempFileAndWriter("stdout")
-    val (stderrFile, stderrWriter) = FileUtil.tempFileAndWriter("stderr")
-    val (commandFile, commandWriter) = FileUtil.tempFileAndWriter("command")
+    val callDirectory = Paths.get(hostExecutionPath(workflowDescriptor).toFile.getAbsolutePath, "call-" + call.name).toFile
+    callDirectory.mkdir()
+
+    val (stdoutFile, stdoutWriter) = FileUtil.tempFileAndWriter("stdout", callDirectory)
+    val (stderrFile, stderrWriter) = FileUtil.tempFileAndWriter("stderr", callDirectory)
+    val (commandFile, commandWriter) = FileUtil.tempFileAndWriter("command", callDirectory)
 
     commandWriter.write(commandLine)
     commandWriter.flushAndClose()
 
-    s"/bin/bash $commandFile" ! ProcessLogger(stdoutWriter writeWithNewline, stderrWriter writeWithNewline)
+    // FIXME this is hardcoded to use Docker for the very short term.  Once we can determine Docker/non-Docker
+    // FIXME then dockerRun can either be what it is here/empty string and hopefully this should all work.
+    val DockerImage = "ubuntu:latest"
+    val hostWorkflowExecutionFile = hostExecutionPath(workflowDescriptor).toFile
+    val containerExecutionDir = s"/root/${hostWorkflowExecutionFile.toString}"
```

Comment:
sanity preservation sounds nice. I suspect it doesn't matter a ton either way though